src/_P164_gases_ens160.ino

```diff
@@ -0,0 +1,158 @@
+#include "_Plugin_Helper.h"
+#ifdef USES_P164
+
+// #######################################################################################################
+// #################################### Plugin-164: Gases - ENS16x (tvoc,eco2) ###########################
+// #######################################################################################################
+// P164 "GASES - ENS16x (TVOC, eCO2)"
+// Plugin for ENS160 & ENS161 TVOC and eCO2 sensor with I2C interface from ScioSense
+// Based upon: https://github.com/sciosense/ENS160_driver
```

Comment:
Based on the [MIT license in that repository](https://github.com/sciosense/ENS160_driver/blob/master/LICENSE.md), the license file(content) should be included with any copy or substantial reproduction, so you need to include that text here for legal reasons.